third_party/blink/renderer/bindings/core/v8/record_replay_interface.cc

```diff
@@ -3130,6 +3126,66 @@ Object.assign(__RECORD_REPLAY__, {
   replayEval
 });
 
+/** ###########################################################################
+ * {@link patchReplayApi} decorates our API objects/functions with extra
+ * diagnostics.
```

Comment:
This comment needs to be clearer about what patchReplayApi does.  A better abstraction could help, like having a function onReplayApiCalled which bumps the counter (and could do other stuff potentially).